.ado/publish.yml

```diff
@@ -1,45 +1,193 @@
-# Publishing VSCode Extensions https://code.visualstudio.com/api/working-with-extensions/publishing-extension
+# Publishing VSCode Extensions: https://code.visualstudio.com/api/working-with-extensions/publishing-extension
 #   * $(PAT): https://stackoverflow.com/questions/58192624/publishing-vs-code-extension-via-azure-devops/58195815#58195815
 #   * Extension Manifest (package.json): https://code.visualstudio.com/api/references/extension-manifest
+# Building Py .whl for multiple Linux distros - manylinux: https://github.com/pypa/manylinux
+# aarch64 (ARM64) support for manylinux builds: https://codepyre.com/2019/12/arming-yourself/
 
 variables:
-  Ver.Major: 0
-  Ver.Minor: 0
-  Ver.Patch: 0
+  VSCodeExtensionBuildPlatform: 'Linux'
+  PlatfIndepPyPackBuildPlatf: 'Linux'
 
 name: $(BuildId)
 
 trigger: none
 pr: none
 
 jobs:
-- job: "Publish"
-  pool: 
-    vmImage: 'ubuntu-latest'
+- job: "Build"
+  strategy:
+    matrix:
+      linux_x64:                        # 2023.06.30 ESRP Publishing of Py packs is not supported for linux_x86_64.
+                                        # The "manylinux" approach is used.
+        imageName: 'ubuntu-latest'      # Manylinux Py (x86_64) + VSCode Ext.
+        arch: x86_64
+      linux_ARM64:
+        imageName: 'ubuntu-latest'      # Manylinux Py (ARM64). Takes 45 - 65 minutes, hence is in a separate job.
+        arch: aarch64
+      mac:
+        imageName: 'macOS-latest'       # MacOS-specific Py (Mac is usually quite limited).
+        arch: x86_64
+      windows:
+        imageName: 'windows-latest'     # Win-specific Py + Platform-independent Py.
+        arch: x86_64
+  pool:
+    vmImage: $(imageName)
+  variables:
+    arch: $(arch)
+  timeoutInMinutes: 90
+
   steps:
   - task: UsePythonVersion@0
     inputs:
       versionSpec: '3.11'
 
   - script: |
       python ./prereqs.py --install
+    displayName: Install Prereqs
+
+  - script: |
       npm install -g @vscode/vsce
-    displayName: Install VSCE Pub Prereqs
-    
+    displayName: Install Prereqs for VSCode Extension
+    condition: and(eq(variables['Agent.OS'], '${{ variables.VSCodeExtensionBuildPlatform }}'), eq(variables['arch'], 'x86_64'))
+   
   - script: |
-      python build.py --wasm --npm --vscode
+      python build.py --wasm --npm --vscode --no-check --no-test --no-check-prereqs
```

Comment:
Are we skipping the checks/tests/prereqs here purely for time saving? They should all pass and only take a min or so, right?

Reply:
I assume so, but will have to run to see.